src/test/packages/external/zarf.yaml

```diff
@@ -0,0 +1,15 @@
+kind: ZarfPackageConfig
+metadata:
+  name: helm-charts
+  description: Example showcasing mutiple ways to deploy helm charts
+  version: 0.0.1
+components:
+- name: demo-helm-charts
+  required: true
+  charts:
+  - name: podinfo
+    version: 6.5.4
```

Comment:
Can we use 6.4.0 instead to keep our podinfo version consistent (it helps with filesystem space considerations in CI)